test/components/Icon.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import Icon from '../../src/Icon';
+
+describe('Icon', () => {
+
+  it('renders correctly', () => {
+    const component = renderer.create(
+      <Icon icon='foo' />
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
This seems to work, but all the [official](http://facebook.github.io/jest/blog/2016/07/27/jest-14.html) [examples](https://facebook.github.io/jest/docs/tutorial-react.html) call `toJSON()` before calling `toMatchSnapshot()`.